src/containers/GalacticPropertiesComboContainer.jsx

```diff
@@ -0,0 +1,203 @@
+import {
+  drawerContainer,
+  galacticPropertiesDrawerContent,
+  linkActive,
+  propertyItem,
+} from '../components/charts/galacticProperties/galactic-properties.module.scss';
+
+class GalacticPropertiesComboContainer extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      data: null,
+      navItems: [],
+      updateDomain: false,
+      properties: [
+        {
+          name: 'Brightness vs Distance',
+          color: 'rgb(254, 216, 40)',
+        },
+        {
+          name: 'Color vs Distance',
+          color: 'rgb(26, 181, 121)',
+        },
+      ],
+      options: {},
+    };
+  }
+
+  componentDidMount() {
+    const { properties } = this.state;
+    const {
+      widget: { source, options },
+    } = this.props;
+    const { multiple } = options || {};
+    const activeProperty = properties[0];
+
+    API.get(source).then(response => {
+      const { data } = response;
+      const responseData = this.getDataObjects(data, multiple);
+
+      this.setState(prevState => ({
+        ...prevState,
+        data: responseData,
+        navItems: this.generateNavItems(properties, activeProperty),
+        options: this.getBrightnessOptions(),
+      }));
+    });
+  }
+
+  getBrightnessOptions = () => {
+    return {
+      title: 'Brightness Vs Distance',
+      xAxisLabel: 'Distance (Billion Ly)',
+      yAxisLabel: 'Total Brightness',
+      xValueAccessor: 'distance',
+      yValueAccessor: 'brightness',
+      tooltipAccessors: ['distance', 'brightness'],
+      tooltipUnits: ['Billion Ly'],
+      tooltipLabels: ['Distance', 'Brightness'],
+      xDomain: [0, 28],
+      yDomain: [0, 200],
+    };
+  };
+
+  getColorOptions = () => {
+    return {
+      title: 'Color Vs Distance',
+      xAxisLabel: 'Distance (Billion Ly)',
+      yAxisLabel: 'Flux ratio i/z (color)',
+      xValueAccessor: 'distance',
+      yValueAccessor: 'color',
+      tooltipAccessors: ['distance', 'color'],
+      tooltipUnits: ['Billion Ly', 'Flux ratio'],
+      tooltipLabels: ['Distance', 'Color'],
+      xDomain: [0, 16],
+      yDomain: [0, 2],
+    };
+  };
+
+  getDataObjects = (data, multiple) => {
+    if (multiple) {
+      return data.map(targetData => {
+        if (!targetData.objects) return null;
+        return targetData.objects;
+      });
+    }
+    return isArray(data.objects) ? data.objects : [];
+  };
+
+  userGalacticPropertiesCallback = data => {
+    console.log({ data }); // eslint-disable-line no-console
+  };
+
+  setActiveProperty = property => {
+    const { properties } = this.state;
+    const loadOptions =
+      property.name === 'Brightness vs Distance'
+        ? this.getBrightnessOptions()
+        : this.getColorOptions();
+    this.setState(prevState => ({
+      ...prevState,
+      navItems: this.generateNavItems(properties, property),
+      options: loadOptions,
+      updateDomain: true,
+    }));
+  };
+
+  generateNavItems(properties, activeScramble) {
```

Comment:
clean up this `activeScramble` variable name.  That was specific to the galaxyScrambler.